interfaces/Portalicious/src/app/pages/projects-overview/create-project-button/create-project-button.component.html

```diff
@@ -0,0 +1,39 @@
+<p-button
+  label="Create new project"
+  i18n-label
+  outlined
+  icon="pi pi-plus"
+  class="text-xs sm:text-base"
+  (click)="formVisible.set(true)"
+/>
+<app-form-sidebar
```

Comment:
Can we change the order? Since the token is a password (or password like), it makes more sense to have that second. So first the AssedId and then the Token